src/Components/Data.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import {
+  Flex,
+  Image,
+  Stack,
+  Text,
+  Heading,
+  Divider,
+  Box,
+  Tooltip,
+  Spacer,
+  useBreakpointValue,
+} from '@chakra-ui/react';
+import { Link } from 'react-router-dom';
+import { StarIcon } from '@chakra-ui/icons';
+
+const Data = ({ data, loading }) => {
+  const skeletonArray = Array.from({ length: 80 });
+  const isMobile = useBreakpointValue({ base: true, md: false });
+
+  return (
+    <Box>
+      <Flex
+        direction={isMobile ? 'column' : 'row'}
```

Comment:
```suggestion
        direction={{base: "column",md:"row"}}
```

chakra have inbuilt responsive values 
please use them

Reply:
done